logstage/logstage-sink-file/src/main/scala/com/github/pshirshov/izumi/models/LogFile.scala

```diff
@@ -0,0 +1,121 @@
+package com.github.pshirshov.izumi.models
+
+import java.nio.file.{FileAlreadyExistsException, Files, Paths, StandardOpenOption}
+import scala.io.Source
+import scala.util.{Failure, Success, Try}
+import LogFile.TryOps._
+
+trait LogFile {
```

Comment:
Interface has too many unused methods. `name`, `path`, `filePath` are only used inside, updateStatus is never used